conda_smithy/templates/build_utils.py.tmpl

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+try:
+    from ruamel_yaml import safe_load, safe_dump
+except ImportError:
+    from yaml import safe_load, safe_dump
+import os
+import subprocess
+import json
+import argparse
+
+global_config = json.loads('''
+{{ channels | tojson }}''')
```

Comment:
Can this be read from the `conda-forge.yml` file instead of a jinja template?

Reply:
After that we don't need to vendor this file in every feedstock

Reply:
yep, that could be done.  Right now we only use that for storing tokens right?